Libraries/LibThread/Mutex.cpp

```diff
@@ -26,34 +26,35 @@
 
 #include "assert.h"
```

Comment:
```suggestion
#include <assert.h>
```

This is a LibC header...